Lib/opcode.py

```diff
@@ -181,14 +181,16 @@ def jabs_op(name, op):
 def_op('MAKE_FUNCTION', 132)    # Flags
 def_op('BUILD_SLICE', 133)      # Number of items
 
-def_op('LOAD_CLOSURE', 135)
```

Comment:
Why the opcode shuffle?

Reply:
My initial impression was that `MAKE_CELL` should be at the front of the cell-related list and I figured it would be better to preserve the gap at 134.  I figured it wouldn't matter but if it's a problem I can put `MAKE_CELL` at 139.